docker-compose.yaml

```diff
@@ -40,27 +21,27 @@ services:
     ports:
       - "127.0.0.1:8080:8080"
     volumes:
-      - ./searxng:/etc/searxng:rw
+      - ./searxng:/etc/searxng
     environment:
       - SEARXNG_BASE_URL=https://${SEARXNG_HOSTNAME:-localhost}/
-    cap_drop:
-      - ALL
-    cap_add:
-      - CHOWN
-      - SETGID
-      - SETUID
-    logging:
-      driver: "json-file"
-      options:
-        max-size: "1m"
-        max-file: "1"
+    security_opt:
+      - no-new-privileges:true
+
+  valkey:
+    container_name: valkey
+    image: cgr.dev/chainguard/valkey:latest
+    command: valkey-server --save 30 1
+    networks:
+      - searxng
+    volumes:
+      - valkey-data:/data
```

Comment:
Maybe could use the existing volume `redis-data` for regaining the existing data from redis?

Reply:
The issue is that the ownership of the RDB file differs between the Redis and Valkey container.

Reply:
Can't you force some UID and GID? So that it's the same as the redis one?

Reply:
It is possible, you can run the container as root as it was originally with Redis and *theoretically* there should be no problem.

Reply:
My point is to not break existing installations. I think just name the volume like before `valkey-data`. The data will be reconstructive anyway.